django_sorcery/db/query.py

```diff
@@ -6,11 +6,48 @@
 import sqlalchemy as sa
 import sqlalchemy.orm  # noqa
 
+from django.db.models.constants import LOOKUP_SEP
+
+from ..utils import lower
 from . import meta
+from .models import Base as ModelBase
 
 
 Operation = namedtuple("Operation", ["name", "args", "kwargs"])
 
+# todo add transforms support - e.g. column__date__gt
```

Comment:
afaik sqla dialects don't have any knowledge on date functions on db side or any db specific date formattings, it all depends on `func` magic basically. you might need to do per dialect things for this.

Reply:
yea will get more complicated. for now will leave as todo so at least its documented shortcoming